plaso/parsers/__init__.py

```diff
@@ -5,6 +5,7 @@
 from plaso.parsers import amcache
 from plaso.parsers import android_app_usage
 from plaso.parsers import apache_access
+from plaso.parsers import apthistory
```

Comment:
Please rename to "apt_history" - and associated files. See "bash_history" for a similarly named parser/formatter etc.